3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/remove_hard_coded_jurisdiction_depth.psql

```diff
@@ -0,0 +1,44 @@
+-- Deploy reveal_irs_zambia_2020:remove_hard_coded_jurisdiction_depth to pg
+-- requires: reveal_database_views:jurisdictions_tree
+
+BEGIN;
+
+-- Add zambia_jurisdictions in pending.
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS pending_:schema .zambia_jurisdictions AS
+SELECT *,
+    jurisdiction_id AS opensrp_jurisdiction_id,
+    CASE
+        WHEN is_leaf_node != TRUE THEN MD5(CONCAT(jurisdiction_id, '_remainder'))
```

Comment:
Do this instead of `WHEN jurisdiction_depth < 5 THEN MD5(CONCAT(jurisdiction_id, '_remainder'))`